### PR TITLE
Improve loading time for patching

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

**Comment by eregon** [comment]
@pitr-ch What option?

File: lib/patches/stdlib/rubygems.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'openssl-stubs' if Truffle::Boot.patching_openssl_enabled?
 
 Truffle::Patching.require_original __FILE__
-Truffle::Patching.install_gem_activation_hook if Truffle::Boot.get_option 'patching'
+Truffle::Patching.install_gem_activation_hook
```
Comment (pitr-ch) [inline lib/patches/stdlib/rubygems.rb]: Sorry I meant: Why is the condition removed here?

Comment (eregon) [reply lib/patches/stdlib/rubygems.rb]: See a778da3cef07878113c5f7eb20116d899ec32bc5:
> This file is only required if patching is enabled.

Comment (pitr-ch) [reply lib/patches/stdlib/rubygems.rb]: ah, ok then.

**Pull request merged**